wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/Scheduler.java

```diff
@@ -195,30 +177,25 @@ public void run() {
     // Get button input (going backwards preserves button priority)
     if (m_buttons != null) {
       for (int i = m_buttons.size() - 1; i >= 0; i--) {
-        ((ButtonScheduler) m_buttons.elementAt(i)).execute();
+        m_buttons.get(i).execute();
```

Comment:
Note that if you want to go backwards with a list iterator, you have to manually set the start index. It defaults to start at index `0` instead of `size - 1`